### ISSUE TITLE
MPI Benchmark json file with cumulative timing and IO

### ISSUE THREAD

**Issue by jaimesouza** [devitocodes/devito#1289]
I executed acoustic benchmark with 20 MPI processes and operator `Forward` ran in 22.90 s.

But each rank generates it's own json file (all generated json files have the same content), line below:

```
{
    "timestamp": "2020-05-15T043236",
    "meta": {},
    "timings": {
        "section0": {
            "gflopss": 90.69606466220392,
            "oi": 34.961945414101685,
            "timings": 443.52664999999996
        },
        "section1": {
            "gflopss": 0.1710404425835923,
            "oi": 21.125,
            "timings": 0.001897000000000009
        },
        "section2": {
            "gflopss": 12.967484899349396,
            "oi": 114.16666666666669,
            "timings": 14.365808999999993
        }
    }
}
```

So, I have 20 json files with exactly the same content (Why redundancy?).

In section0, the `timings` is 443.5 s and `oi` is 34.9, which is comulative for all 20 processes.

**Comment by mloubout** [comment]
Isn't this fixed @jaimesouza

**Comment by jaimesouza** [comment]
@mloubout It isn't yet. I am working on a PR to fix that.

**Comment by georgebisbas** [comment]
This issue can be now closed as PR #1295 is merged/
